roles/init_pd_topo/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: generate init_pd_topo script
+  template:
+    src: "init_pd_topo.sh.j2"
+    dest: "{{ playbook_dir }}/scripts/init-pd-topo.sh"
```

Comment:
Should this file being ignored?

Reply:
It can't be ignored, it will be executed during `start, rolling_update` operation.

Reply:
I mean should we put it in `.gitignore` since it is generated?

Reply:
This script is generated when the cluster is deployed. I think it does not need to be added to `.gitignore`